tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,22 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 27, 2022
+
+**General changes**
+
+* Support joining multiple organizations by invitation.
+
+    In the TiDB Cloud console, you can view all organizations you have joined and switch them. For details, see [Switch organizations](/tidb-cloud/manage-user-access.md#switch-organizations).
```

Comment:
```suggestion
    In the TiDB Cloud console, you can view all organizations you have joined and switch between them. For details, see [Switch organizations](/tidb-cloud/manage-user-access.md#switch-organizations).
```

Reply:
Please also check and update related descriptions in `manage-user-access.md#switch-organizations`.